test/test_output_engine.py

```diff
@@ -15,34 +15,57 @@
 class TestOutputEngine(unittest.TestCase):
     """ Test the OutputEngine class functions """
 
-    MOCK_OUTPUT = {
-        "modulename0": {
-            "1.0": {"CVE-1234-1234": "MEDIUM", "CVE-1234-9876": "LOW"},
-            "2.8.6": {"CVE-1234-1111": "LOW"},
+    MOCK_OUTPUT = [
+        {
+            "vendor": "vendorname0",
+            "product": "modulename0",
```

Comment:
We should also replace "modulename0" with "productname0". So, it won't create confusion for new comers.

Reply:
Thanks, Nice Catch